content/blog/2018-04-23-noo-noo-from-collyers-college-wins-sourcebots-2018.md

```diff
@@ -0,0 +1,82 @@
+---
+title: “Noo Noo” from Colleyr's College wins SourceBots 2018
+date: 2018-04-23
+---
+
+SourceBots 2018 reached its exciting climax this weekend as the teams from
+across the south coast gathered at the University of Southampton for the
+competition event. The two-day event saw their robots compete in forty matches
+to determine which would be crowned champion in this year's game
+_Pirate Islands_.
+
+### Competition
+
+The game required the robots to remove opposing tokens from their scoring zone
+and collect their own tokens from the opposing side of the 8×8m arena. As any of
+the four robots in the arena could move any token, this required the robots to
+detect which tokens were where and to react to changes.
+
+The matches were broken into two main sections. First was a league stage with
+matches spaced throughout the weekend and interspersed with "tinker time" which
+both allowed the teams to test and improve their robots as well as gauge the
+strengths of their opponents. The time between rounds proved invaluable to many
+of the teams and we saw the robots improve greatly over the course of the
+weekend.
+
+Once the league was complete, the results from the league seeded a knockout
+stage. While every robot entered the knockouts, it all came down to the final
+match between _[Noo Noo][CLY-robot-image]_ from [Collyer's][collyers] team
```

Comment:
I don't believe the robot was called _Noo Noo_. The comp data shows _SCC_ as a team name? Is the `name` meant to be team name or robot name?

Reply:
I'm basing this off the flavour document linked in Slack.

Reply:
Just confirmed with the team via Discord that the robot is called Noo Noo. SCC was their team name

Reply:
I suggest that we consider that Robot and Team names may be different in the future. How difficult would it be to add this functionality to `srcomp` @PeterJCLaw Does the solution to this problem justify the effort required to add the property to the team object?